server/src/PendingCeremonies.hs

```diff
@@ -8,13 +8,13 @@
 -- | This module implements a secure and efficient way to temporarily store
 -- webauthn options for the create/register and get/login webauthn operations
 -- along with generating the challenge needed for them.
-module PendingOps
-  ( PendingOpsConfig (..),
-    defaultPendingOpsConfig,
-    newPendingOps,
-    PendingOps,
-    insertPendingOptions,
-    getPendingOptions,
+module PendingCeremonies
+  ( PendingCeremonyConfig (..),
+    defaultPendingCeremonyConfig,
```

Comment:
Should be `PendingCeremoniesConfig` and `defaultPendingCeremoniesConfig`, since it's not the ceremony that's being configured, but the `PendingCeremonies` type.